daily-patch-information/src/main/java/org/wso2/patchinformation/MainEmailSender.java

```diff
@@ -0,0 +1,132 @@
+//
+// Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+package org.wso2.patchinformation;
+
+import org.apache.log4j.Logger;
+import org.wso2.patchinformation.email.EmailContentCreator;
+import org.wso2.patchinformation.email.EmailSender;
+import org.wso2.patchinformation.exceptions.PatchInformationException;
+import org.wso2.patchinformation.jira.JIRAAccessor;
+import org.wso2.patchinformation.jira.JIRAIssue;
+import org.wso2.patchinformation.pmt.PmtAccessor;
+import org.wso2.patchinformation.properties.PropertyValues;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import static org.wso2.patchinformation.constants.EmailConstants.EMAIL_SUBJECT_CUSTOMER;
+import static org.wso2.patchinformation.constants.EmailConstants.EMAIL_SUBJECT_INTERNAL;
+import static org.wso2.patchinformation.constants.EmailConstants.MAIN_HEADER_CUSTOMER;
+import static org.wso2.patchinformation.constants.EmailConstants.MAIN_HEADER_END;
+import static org.wso2.patchinformation.constants.EmailConstants.MAIN_HEADER_INTERNAL;
+/**
+ * Sends 2 emails on behalf of the engineering efficiency team. One on Customer related JIRA issues,
+ * and the other on Proactive JIRA issues and their corresponding patch information.
+ */
+public class MainEmailSender {
+
+    private static final Logger LOGGER = Logger.getLogger(MainEmailSender.class);
+
+    public static void main(String[] args) {
+        try {
+            LOGGER.info("Executing process to send email on Internal JIRA issues.");
+            executeEmailSendingProcess(false);
```

Comment:
Instead of making the methods as static and calling with class reference, please make them as non static methods. And then make an instance (new EmailSender()) to invoke those methods.